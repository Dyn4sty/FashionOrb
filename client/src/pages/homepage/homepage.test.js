import React from "react";
import { shallow } from "enzyme";
import HomePage from "./homepage";

it("should render HomePage component", () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
