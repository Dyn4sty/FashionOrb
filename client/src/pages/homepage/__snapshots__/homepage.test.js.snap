// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage component 1`] = `
<styled.div>
  <BannerItem
    background="https://i.ibb.co/nP6R16G/S1.jpg"
    bannerheight="680px"
    bannertype="center"
  >
    <div
      className="flex_arrange"
    >
      <p>
        New Arrivals
      </p>
      <h1>
        Enjoy This 
        <br />
         Summer Trends
      </h1>
      <p>
        On Eligible Items in order of $100 or more
      </p>
      <CustomButton
        as={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "displayName": "Link",
            "propTypes": Object {
              "innerRef": [Function],
              "onClick": [Function],
              "replace": [Function],
              "target": [Function],
              "to": [Function],
            },
            "render": [Function],
          }
        }
        to="/shop"
      >
        Shop Now
      </CustomButton>
    </div>
  </BannerItem>
  <Directory />
  <styled.div>
    <BannerItem
      background="https://i.ibb.co/vBBdqxq/slider1-hp4.jpg"
      bannerheight="400px"
      bannertype="left"
      className="bannerItem"
    >
      <div>
        <h1>
          New 
          <br />
           Arrival
        </h1>
        <p>
          On Eligible Items in stock for now
        </p>
      </div>
    </BannerItem>
    <BannerItem
      background="https://i.ibb.co/V9P7SDS/slider3-hp4.jpg"
      bannerheight="400px"
      bannertype="left"
      className="bannerItem"
    >
      <div>
        <h1>
          Summer 
          <br />
           Sales Off
        </h1>
        <p>
          On Eligible Items off pricess in stock
        </p>
      </div>
    </BannerItem>
  </styled.div>
</styled.div>
`;
